pkg/authorization/authorizer/authorizer.go

```diff
@@ -158,7 +161,7 @@ func (a *openshiftAuthorizer) getEffectivePolicyRules(namespace string, user use
 		}
 	}
 
-	return effectiveRules, nil
+	return effectiveRules, kerrors.NewAggregate(errs)
```

Comment:
returning items that are valid to use along with an error is weird enough that it merits a function godoc